packages/provider/src/transports/base-provider-transport.ts

```diff
@@ -49,7 +49,7 @@ export abstract class BaseProviderTransport implements ProviderTransport {
     throw new Error('abstract method')
   }
 
-  openWallet(path?: string, state?: any, defaultNetworkId?: string | number) {
+  openWallet(path?: string, state?: OpenWalletState, defaultNetworkId?: string | number) {
```

Comment:
all looks good, perhaps we should rename the variable name from `state` to `intent` and then `OpenWalletState` -> `OpenWalletIntent` ?